tests/periodic_quarter_shell.prm

```diff
@@ -0,0 +1,93 @@
+# testing for periodic conditions in phi 
+# for 2D quarter spherical shell geometry
+# based on boundary_velocity_function_spherical.prm
+
```

Comment:
You will also need to add an output directory for this test. You can read about how to do that here: http://www.math.clemson.edu/~heister/manual.pdf#sec%3Awriting_tests, let us know if you have questions about that.